hw2/05-form.js

```diff
@@ -1 +1,52 @@
-// Add your code here
+const form = document.getElementById('form-exercise');
+
+form.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event) {
+  const name = document.getElementById('name');
+  const email = document.getElementById('email');
+  const registration = document.getElementById('registration');
+  const previousClasses = document.getElementById('boxes');
+  const checkboxes = previousClasses.querySelectorAll('input[type="checkbox"]');
+  const anythingElse = document.getElementById('anything-else');
+  const noSubmission = 'no submission';
+
+  if (!name.value && !email.value) {
+    console.warn('You must enter some data to submit this form');
+  } else {
+    console.group('========= Form Submission =========');
+
+    console.log('Name: ', name.value ? name.value : noSubmission);
+    console.log('Email: ', email.value ? email.value : noSubmission);
+
+    if (registration.value && registration.value != 'Choose an Option') {
+      console.log('Registration: ', registration.value);
```

Comment:
Caterina suggested in class that instead of concatenating string which I think this is doing that she suggests using string interpolation so 
- "console.log('Registration: ', registration.value);"
   would turn into 
"console.log(`Registration: ${registration.value}`);"

git is removing my backtics on line just above this so, keep in mind there is a opening backtic before the word registration and after the last curly brace

Reply:
This method is not technically concatenation but your feedback is helpful, thank you!